contracts/LiquidityBridgeContractV2.sol

```diff
@@ -58,7 +58,6 @@ contract LiquidityBridgeContractV2 is Initializable, OwnableUpgradeable, Reentra
     event PegoutCollateralIncrease(address from, uint256 amount);
```

Comment:
As it's a new implementation, even that is jus a small fix, and the V2 is deployed, we should create a V2_1 file and do the changes over there.

Reply:
I don't agree with that point because of the following reasons:
- We did it for V1 and it only introduced noise, PRs were harder to review both for us and for external teams (security) since the real changes were hidden among the rest of the code and git didn't provide any way to identify the changes since its a new file
- We're not maintaining several versions of the protocol at the same time. So keeping older versions only increases the number of files on the repo that we need to take care about
- If for some reason we need to check older versions, we are already using git as versioning tool. If you want to check the deployed code of the latest version you can refer to [this tag](https://github.com/rsksmart/liquidity-bridge-contract/releases/tag/v1.1.2)
- The versioning of the files is not accurate, we're on LBC V2 however there is not breaking change between the V1 and V2, only minor increments

Which would be the rationale for generating new files for each version? Because I don't mind doing it but I need a reason since that approach didn't bring value in the last deploy